packages/material-ui/src/ImageListItemBar/ImageListItemBar.test.js

```diff
@@ -21,7 +21,7 @@ describe('<ImageListItemBar />', () => {
   }));
 
   const itemData = {
-    img: 'images/image-list/00-52-29-429_640.jpg',
+    img: 'fake.png',
```

Comment:
Should the imports be absolute or relative?
```suggestion
    img: '/fake.png',
```

Reply:
Let's make them absolute to be safe.

Reply:
done